test/scenarios-bootc/cos9-src@bootc-standard2.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-bootc.ks.template cos9-bootc-source
+    launch_vm host1 centos9 "" "" "" "" "" "" "1"
+
+    # Open the firewall ports. Other scenarios get this behavior by
+    # embedding settings in the blueprint, but there is no blueprint
+    # for this scenario. We need do this step before running the RF
+    # suite so that suite can assume it can reach all of the same
+    # ports as for any other test.
+    configure_vm_firewall host1
```

Comment:
Can't this be done in the containerfile?
Found example where ansible is executed during build to configure firewall: https://gitlab.com/CentOS/cloud/centos-bootc-examples/-/tree/main/ansible-firewalld

Reply:
It can be done in the containerfile, but it will then be part of each containerfile because we do not have templating for them yet. 
I can move it there if you think it's critical for the first implementation.

Reply:
Nah, we can leave it like this for now, but in future I think it'd be better to change it. Like in production I'd have this in the Containerfile so the image is complete (boots and runs).
To reduce repetition we can build an intermediate image (it can be build locally on each job without pushing to registry) and use it in FROM in other images.